temporal-sdk/src/main/java/io/temporal/internal/replay/ReplayWorkflowRunTaskHandler.java

```diff
@@ -286,10 +285,10 @@ private void processLocalActivityRequests(long startTimeNs) throws InterruptedEx
         // much sense. I believe we should add ScheduleToStart timeout for the local activities
         // as well.
```

Comment:
Todo now addressed?

Reply:
Nope! https://github.com/temporalio/sdk-java/issues/1512 Will be in a separate PR.